docs: fix graphic in safari

crisbeto opened the pull request:
Fixes that the "Loved by millions" graphic doesn't look correctly on Safari.

Note that this fix is somewhat repetitive, but I couldn't find a better one. These are the other things I tried:
* Setting units on the size of `#angular-logo`.
* Setting the size of `#angular-logo` inside the `style` tag.
* Setting the size with inline styles.

Fixes #43498.

mary-poppins commented:
You can preview 8e91b6c at https://pr43502-8e91b6c.ngbuilds.io/.

gkalpak commented on aio/content/images/marketing/home/loved-by-millions.svg:
```diff
@@ -5,7 +5,7 @@
   <title>Loved by Millions</title>
   <defs>
-    <symbol id="angular-logo" viewBox="0 0 200 200" width="32" height="32">
+    <symbol id="angular-logo" viewBox="0 0 200 200">
```
I don't have access to "real" Safari, but by playing in an emulator another option that I think would work (and avoid repetition would be to use `<g>` element and a CSS scale transform:

```diff
   <style>
     .land { fill: #f2f2f2; }
     .water { fill: #ddd; }
+    #angular-logo { transform: scale(0.16); }
   </style>
   <title>Loved by Millions</title>
   <defs>
-    <symbol id="angular-logo" viewBox="0 0 200 200">
+    <g id="angular-logo">
       <!-- ... -->
-    </symbol>
+    </g>
```

##
Either way is not great, so leaving it up to you 😇

crisbeto replied:
Nice, that is a bit cleaner 👍

gkalpak reviewed (approved):
Thx for fixing this!
<sub>(And "thank you", Safari 😞)</sub>

Reviewed-for: global-docs-approvers

mary-poppins commented:
You can preview 3702ec1 at https://pr43502-3702ec1.ngbuilds.io/.

gkalpak commented on aio/content/images/marketing/home/loved-by-millions.svg:
```diff
@@ -136,40 +140,41 @@
-  <use href="#angular-logo" x="958" y="100" />
-  <use href="#angular-logo" x="1151" y="85" />
-  <use href="#angular-logo" x="1096" y="518" />
+  <!-- width/height needs to be set on each `use` due to a Safari issue (see #43498). -->
```
This comment is not redundant :grin:

crisbeto committed:
docs: fix graphic in safari

Fixes that the "Loved by millions" graphic doesn't look correctly on Safari.

Note that this fix is somewhat repetitive, but I couldn't find a better one. These are the other things I tried:
* Setting units on the size of `#angular-logo`.
* Setting the size of `#angular-logo` inside the `style` tag.
* Setting the size with inline styles.

Fixes #43498.
